packages/transform-to-vanilla/src/transform-keys/simple-pseudo-selectors.ts

```diff
@@ -0,0 +1,57 @@
+// == Type ================================================================
+type DefaultType = string & NonNullable<unknown>;
+type PseudoSelectorsSign = `_${string}` | `__${string}`;
+type ReplacedPseudoSelectorsSign = `:${string}` | `::${string}`;
+type InputKeyValue = DefaultType | PseudoSelectorsSign;
+type ReturnKeyValue = DefaultType | ReplacedPseudoSelectorsSign;
+
+// == Interface ================================================================
+export function replacePseudoSelectors(keyStr: InputKeyValue): ReturnKeyValue {
+  const kebabKeyStr = camelToKebab(keyStr);
+
+  return hasSinglePseudoSelector(kebabKeyStr)
+    ? `:${kebabKeyStr.substring(1, kebabKeyStr.length)}`
+    : hasDoublePseudoSelector(kebabKeyStr)
+    ? `::${kebabKeyStr.substring(2, kebabKeyStr.length)}`
+    : kebabKeyStr;
+}
+
+// == Utils ====================================================================
+function camelToKebab(camelCase: InputKeyValue) {
+  return camelCase.replace(/[A-Z]/g, "-$&").toLowerCase();
+}
+function hasSinglePseudoSelector(
+  value: InputKeyValue
+): value is PseudoSelectorsSign {
+  return value.startsWith("_") && !value.startsWith("__");
+}
+function hasDoublePseudoSelector(
+  value: InputKeyValue
+): value is PseudoSelectorsSign {
+  return value.startsWith("__");
+}
+
+// == Tests ====================================================================
+if (import.meta.vitest) {
+  const { describe, it, expect } = import.meta.vitest;
+
+  describe.concurrent("Replace Simple Pseudo Selectors Sign", () => {
+    it("No Simple Pseudo Selectors Sign", () => {
+      expect(replacePseudoSelectors("after")).toBe("after");
+    });
```

Comment:
Please add another test case.

- `backgroundColor` should be same
- `.myClassName > &` should be same
- `&:hover:not(:active)` should be same
- `@supports (display: grid)` should be same

In the current code, it is converted to cabab and an error is expected.